src/EPR.Accreditation.API/Repositories/Interfaces/IRepository.cs

```diff
@@ -72,5 +72,7 @@ public interface IRepository
             DTO.SaveAndComeBack saveAndContinue);
 
         Task<IEnumerable<DTO.Material>> GetMaterials();
+
+        Task SetOverseasAgent(Guid id, bool? hasOverseasAgent);
```

Comment:
Again, setting of the overseas agent should be done on update to the accreditation table.